packages/wrangler/src/vectorize/client.ts

```diff
@@ -2,36 +2,32 @@ import { fetchListResult, fetchResult } from "../cfetch";
 import { requireAuth } from "../user";
 import type { Config } from "../config";
 import type {
-	VectorizeDistanceMetric,
+	VectorFloatArray,
+	VectorizeAsyncMutation,
 	VectorizeIndex,
 	VectorizeIndexDetails,
+	VectorizeMatches,
+	VectorizeMetadataIndexList,
+	VectorizeMetadataIndexProperty,
+	VectorizeMetadataIndexPropertyName,
 	VectorizeQueryOptions,
 	VectorizeVector,
 	VectorizeVectorMutation,
-} from "@cloudflare/workers-types";
+} from "./types";
 import type { FormData } from "undici";
 
 const jsonContentType = "application/json; charset=utf-8;";
 
-interface VectorizeIndexResult extends VectorizeIndexDetails {
-	readonly created_on: string;
-	readonly modified_on: string;
-	readonly config: IndexConfigResult;
-}
-
-interface IndexConfigResult {
-	metric: VectorizeDistanceMetric;
-	dimensions: number;
-}
-
 export async function createIndex(
 	config: Config,
-	body: object
-): Promise<VectorizeIndexResult> {
+	body: object,
+	deprecatedV1: boolean
+): Promise<VectorizeIndex> {
 	const accountId = await requireAuth(config);
+	const version_param = deprecatedV1 ? `` : `/v2`;
```

Comment:
super nit (that's not worth a commit needing a rebuild): in js, camelCase is preferred

Reply:
Fixed it along with some other changes!